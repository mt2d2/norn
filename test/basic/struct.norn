struct Test
	i: Int
	b: Boolean
end

def main(): Void
	# tester: Test = 8

	tester: Test = new(9)
	tester.i = 8

	test: Int = tester.i

	print("Value of struct: ")
	println(tester.i)
end
