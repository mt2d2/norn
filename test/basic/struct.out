Value of struct: 8
Value of struct: true
